pricegraph/src/orderbook.rs

```diff
@@ -49,11 +54,19 @@ impl Orderbook {
         let mut orders = OrderCollector::default();
         let mut users = UserMap::default();
 
+        const MIN_AMOUNT_U128: u128 = MIN_AMOUNT as _;
+        const MIN_AMOUNT_U256: U256 = U256([MIN_AMOUNT as _, 0, 0, 0]);
+
         for element in elements {
```

Comment:
Is "element" different from "order" somehow? Is it a touched order? Maybe subject of a different PR, but maybe we use a more descriptive term here?

Reply:
`Element` is an auction element (let me know if I should add more to the docs):
https://github.com/gnosis/dex-services/blob/1a0d8c90ddac3decd0dd449237da254fa783616d/pricegraph/src/encoding.rs#L46

`Order` is an internal order representation that is optimized for the algorithm.